crates/typst/src/engine.rs

```diff
@@ -186,6 +308,9 @@ impl Sink {
     }
 
     /// Extend from another sink.
+    /// Using `Sink::extend_tracked` is preferable as it avoids a call to this
```

Comment:
A general nit about comment style: When a line is ended early due to a conceptual break (i.e. after "Extend from another sink" there should be a blank line as it's a new paragraph). Same above actually with "Later, during warning suppression [...]".

So either it is close enough to wrap at 80 columns or it should be a paragraph break.